config/vufind/config.ini

```diff
@@ -616,11 +616,14 @@ username_hint = username_only_letters_numbers_and_basic_punctuation
 ; Persistent login token lifetime in days
 ;persistent_login_lifetime = 60
 
+; Whether to send email warnings about suspicious logins. Default is true.
+;send_persistent_login_warnings = true
+
 ; Subject for the email sent after a suspicious login has been detected.
 ; Default email subject will be the translation for
-; "login_warning_email_subject" and any custom value used here
+; "persistent_login_warning_email_subject" and any custom value used here
 ; will also be run through the translator.
-;login_warning_email_subject = "Alert: Suspicious login detected"
+;persistent_login_warning_email_subject = "Alert: Suspicious login detected"
```

Comment:
Should we change the example to match the actual revised string in the language file? If not, I'd suggest saying "login attempt" rather than "login" here.